packages/amplify-container-hosting/src/lib/ElasticContainer/file-uploader.js

```diff
@@ -0,0 +1,25 @@
+const fs = require('fs-extra');
```

Comment:
file path is `.../src/lib/Elastic...`. It seems odd to have a lib folder in src.

Reply:
Old code used `lib` at the root level of the package and since I did not want to change the actual code, I just lifted in shifted it inside `src`.